spec/formatters_spec.lua

```diff
@@ -170,5 +189,8 @@ describe("Test Formatters", function()
     assert.spy(s).was.called(1)
     assert:remove_formatter(s)
   end)
-
 end)
+
+-- Local variables:
+-- lua-indent-level: 2;
+-- end:
```

Comment:
seems like some tooling related thing. Can you please remove this?

Reply:
Gone.